frontend/src/app/components/FeaturedExperiment/FeaturedStatus.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React from 'react';
+import { Localized } from 'fluent-react/compat';
+
+const ONE_DAY = 24 * 60 * 60 * 1000;
+const ONE_WEEK = 7 * ONE_DAY;
+const MAX_JUST_LAUNCHED_PERIOD = 2 * ONE_WEEK;
+const MAX_JUST_UPDATED_PERIOD = 2 * ONE_WEEK;
+
+type FeaturedStatusProps = {
+  enabled: boolean,
+  experiment: Object,
+  getExperimentLastSeen: Function
+}
+
+export default class FeaturedStatus extends React.Component {
+  props: FeaturedStatusProps
+
+  render() {
+    const { enabled } = this.props;
+    const launched = this.justLaunched();
+    const updated = this.justUpdated();
+    const showIcon = (enabled || launched || updated);
+
+    return (
+        <div className="featured-status">
+          {showIcon && <div className="star-icon"></div>}
+
+          {launched && <Localized id="experimentListJustLaunchedTab">
+            <div className="tab just-launched-tab">Just Launched</div>
+          </Localized>}
+
+          {updated && <Localized id="experimentListJustUpdatedTab">
+            <div className="tab just-updated-tab">Just Updated</div>
+          </Localized>}
+
+          {enabled && <Localized id="experimentListEnabledTab">
+            <div className="tab enabled-tab">Enabled</div>
+          </Localized>}
+        </div>
+    );
+  }
+
+  justUpdated() {
```

Comment:
Are these bits of just updated / launched logic duplicated from the experiment page? Would be good to pull them out into a util lib to DRY things up

Reply:
They are indeed, I wasn't quite sure where to put the `justUpdated` and `justLaunched` methods.